src/Compilers/Core/Portable/Symbols/ISymbolInternal.cs

```diff
@@ -27,6 +27,8 @@ internal interface ISymbolInternal
 
         string MetadataName { get; }
 
+        int MetadataToken { get; }
```

Comment:
Why is this new declaration needed?

Reply:
Used in DefinitionMap.GetInitialMetadataHandle